internal/bind/params.go

```diff
@@ -177,63 +244,82 @@ func toYdbParam(name string, value interface{}) (table.ParameterOption, error) {
 func Params(args ...interface{}) (params []table.ParameterOption, _ error) {
```

Comment:
Signature of the function was changed in main branch, it needs to rebase the branch and resolve conflicts.